Yona/Yona/Friends/FriendsDashBoardViewController.swift

```diff
@@ -144,64 +120,93 @@ class FriendsDayViewController: MeDashBoardMainViewController {
                 let data = rightTabData[section].activity[theTableView.indexPathForSelectedRow!.row]
                 controller.initialObject = data
                 controller.buddy = buddyToShow
-                
             }
         }
-
     }
     
     override func tableView(_ tableView: UITableView, didSelectRowAtIndexPath indexPath: IndexPath) {
-//        if selectedTab == .right {
-//            performSegueWithIdentifier(R.segue.friendsDayViewController.showFriendsDetailWeek, sender: self)
-//        }
-        
         if selectedTab == .left {
-            performSegue(withIdentifier: R.segue.friendsDayViewController.showFriendsDetailDay, sender: self)
+            performSegue(withIdentifier: R.segue.friendsDashBoardViewController.showFriendsDetailDay, sender: self)
         }
     }
-
     
     // MARK: - Data loaders
-    
     override func loadActivitiesForDay(_ page : Int = 0) {
```

Comment:
This function is really long. Can't you break it apart?

Reply:
Yes, I can break into small method.